File: 1406-subtract-the-product-and-sum-of-digits-of-an-integer/1406-subtract-the-product-and-sum-of-digits-of-an-integer.java
class Solution {
    public int subtractProductAndSum(int n) {
        int prod=1;
        int sum1=0;
        while(0<n)
        {
            prod=prod*(n%10);
            sum1=sum1+(n%10);
            n=n/10;
        }
        return prod-sum1;        
    }
}

File: 1406-subtract-the-product-and-sum-of-digits-of-an-integer/1406-subtract-the-product-and-sum-of-digits-of-an-integer.py
class Solution:
    def subtractProductAndSum(self, n: int) -> int:
        prod=1
        sum1=0
        while 0<n:
            prod=(n%10)*prod
            sum1=(n%10)+sum1
            n=int(n/10)
        return prod-sum1
        